static/js/controllers/contacts.js

```diff
@@ -97,7 +85,15 @@ var _ = require('underscore'),
           actualFilter = $scope.filters;
         }
 
-        Search('contacts', actualFilter, options).then(function(contacts) {
+        var extensions = {};
+        if ($scope.lastVisitedDateExtras) {
+          extensions.displayLastVisitedDate = true;
+        }
+        if ($scope.sortDirection === 'lastVisitedDate') {
+          extensions.sortByLastVisitedDate = true;
+        }
+
+        return Search('contacts', actualFilter, options, extensions).then(function(contacts) {
```

Comment:
Options gets passed around everywhere, and I really didn't want to mix them. I am trying to keep IN stuff separate, because a) I don't want to give the impression that we support this stuff as a first class feature, and b) I want to make it easy to remove if the IN doesn't prove that it's useful